tests/Microsoft.Identity.Test.Integration.netstandard/Microsoft.Identity.Test.Integration.NetStandard.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+
+    <Compile Include="../Microsoft.Identity.Test.Integration.NetFx/**/*.cs" Exclude="../Microsoft.Identity.Test.Integration.NetFx/obj/**/*" />
+    <None Include="../Microsoft.Identity.Test.Integration.NetFx/federationMetadata.xml" />
+
+    <!-- Force pick the netstandard version of our assemblies -->
+    <ProjectReference Include="..\..\src\client\Microsoft.Identity.Client.Broker\Microsoft.Identity.Client.Broker.csproj" AdditionalProperties="TargetFramework=netstandard2.0" />    
```

Comment:
The most subtle change is here - this is how to force the test project to use msal.netstnandard